### PR TITLE
Upload demo app to runway

### PR DESCRIPTION
**Pull request by marianeum**
We're now publishing the demo app on Runway, rather than App Center with app center shutting down next year.

+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -84,18 +84,14 @@ jobs:
...
-          token: ${{secrets.APP_CENTER_TOKEN}}
-          group: Everyone
-          file: app/build/outputs/apk/oss/release/app-${{ env.flavour }}-release.apk
-          releaseNotes: Version ${{github.event.release.tag_name}}
-
-  SupernovaPublish:
-    runs-on: ubuntu-22.04
-    environment: Publishing
-    steps:
       - name: Update Supernova docs
```
Comment (marianeum) [inline .github/workflows/release.yml:87]: removed the separate pipeline for the supernova docs. this means we'll only get prompted once about approving the release

File: README.md [hunk 2]
```diff
@@ -14,10 +14,6 @@ Backpack for Android supports two targets: **Android View** system and **Jetpack
 ### Demo application
 The Backpack demo application is a good way of referring to the variants available for a component and their correct usage.
 The code is available under `/app` directory.
-[The app can be downloaded from App Center](https://install.appcenter.ms/orgs/backpack/apps/backpack-android/distribution_groups/everyone)
```
Comment (marianeum) [inline README.md:17]: keeping the instructions out of here as it won't be public

File: app/build.gradle [hunk 3]
```diff
@@ -39,7 +39,7 @@ android {
     defaultConfig {
         applicationId 'net.skyscanner.backpack'
         versionCode 1
-        versionName "🐨"
+        versionName "1.0.0"
```
Comment (marianeum) [inline app/build.gradle:42]: runway does not support the koala emoji version 😢

File: app/build.gradle [hunk 4]
```diff
@@ -64,11 +64,9 @@ android {
     productFlavors {
         oss {
             dimension "version"
-            versionNameSuffix "-oss"
```
Comment (marianeum) [inline app/build.gradle:67]: just removing the unnecessary suffix

**Review by xo0ps — APPROVED** [review]

**Pull request merged**